lab-24-chris/app/test/service-test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const angular = require('angular');
+require('angular-mocks');
+// const playerService = require('../service/player-service');
+
+describe('testing ngAdventure', function() {
+  beforeEach(() => {
+    angular.mock.module('ngAdventure');
+    // angular.mock.inject(($rootScope, $controller) => {
```

Comment:
does this process make sense? using mock to inject an instance of your custom controller to the angular's scope object?